pkg/apis/kubeone/types.go

```diff
@@ -177,8 +177,8 @@ type CloudProviderSpec struct {
 	Hetzner *HetznerSpec `json:"hetzner,omitempty"`
 	// Openstack
 	Openstack *OpenstackSpec `json:"openstack,omitempty"`
-	// Packet
-	Packet *PacketSpec `json:"packet,omitempty"`
+	// Equinix Metal
+	EquinixMetal *EquinixMetalSpec `json:"equinixMetal,omitempty"`
```

Comment:
We should be consistent in terms of the names we use. We use `equinixMetal` in the API, but in other places, we use `equinixmetal`. Please make sure that we use the same name everywhere, otherwise, it's can be confusing and error-prone.

Reply:
I completely agree and that was something that i was worried about as well. So we agree with calling it `equinixmetal` everywhere right?